sys/xtimer/Makefile.dep

```diff
@@ -0,0 +1,25 @@
+ifneq (,$(filter xtimer,$(USEMODULE)))
```

Comment:
This change was not menat to be here, but as its never included it did not change anything